src/main/webapp/script.js

```diff
@@ -216,3 +216,25 @@ function getLink(schoolName) {
       'school.main_campus,school.institutional_characteristics.level')
 }
 
+/** Adds comments to page. */
+function loadComments() {
+  fetch('/data').then(response => response.json()).then((comments) => {
+    const commentListElement = document.getElementById('comments-container');
+    comments.forEach((comment) => {
+      commentListElement.appendChild(createCommentElement(comment.name,
+          comment.message, comment.time));
+    });
+  });
+}
+
+/**
+ * @param {string} name The name of the user who commented.
+ * @param {string} message The message body of a comment post.
+ * @param {string} time The time of a comment post.
+   @return {HTMLLIElement} commentElement.
```

Comment:
Start the line with a "*".

Reply:
Also, while @param needs the param name, @return just takes an optional description of the returned element. For you, maybe it's `@return {!HTMLLIElement} A comment list item. Be sure to add the "!" which shows that it can't be null. This is a bit confusing - it's not needed for all primitive JavaScript types, which are `string`, `number, and `boolean`, since they're nonnullable by default. Everything else must have ! for nonnullable or ? for nullable.